static/profile.css

```diff
@@ -0,0 +1,24 @@
+body {
+    background-image:url(https://media.istockphoto.com/photos/medium-golden-brown-wood-texture-background-picture-id513694258?k=20&m=513694258&s=170667a&w=0&h=1MRpB5RJmf5ZNa_ap2jr80Udoqcla6EkbG6eMzZ6Z1k=);
```

Comment:
You should probably download the image into the static folder and link it here, just for clarity (and to prevent the page from having to go that url repeatedly to load the background)